### PR TITLE
emds: add preliminary timings for rram

### PR DESCRIPTION
**Pull request by alxelax**
Commit adds preliminary writing timings for flash driver based on rram.

### CODE REVIEW

File: subsys/emds/Kconfig [hunk 1]
```diff
@@ -37,11 +37,15 @@ config EMDS_THREAD_PRIORITY
...
 	int "Time to write one word into flash"
+	default 210 if SOC_FLASH_NRF_RRAM
 	default 41
 	help
 	  Max time to write one word (4 bytes) in flash (in microseconds). This
 	  value is dependent on the chip used, and should be checked against the
 	  chip datasheet.
+	  For rram based flash driver, data is written by blocks of 32 bytes over
+	  buffered writing. This value is used to calculate the time to write
+	  one 32 bytes block.
```
Comment (mia-ko) [inline subsys/emds/Kconfig:48]:
```suggestion
	  For RRAM-based flash drivers, data is written in blocks of 32 bytes over
	  buffered writing. This value is used to calculate the time to write
	  one 32-byte block.
```

File: subsys/emds/Kconfig [hunk 2]
```diff
@@ -63,6 +68,8 @@ config EMDS_FLASH_TIME_BASE_OVERHEAD_US
 	   worst case scenario, before starting storing entries. This value
 	   is dependent on the chip used, and should be checked against the chip
 	   datasheet.
+	   For rram based flash driver, the time is defined by the erasing block size.
+	   The default erase block size is 4096 bytes.
```
Comment (mia-ko) [inline subsys/emds/Kconfig:72]:
```suggestion
	   For RRAM-based flash drivers, the time is defined by the erasing block size.
	   The default erasing block size is 4096 bytes.
```

**Review by mia-ko — CHANGES_REQUESTED** [review]
Do we need a changelog entry for this?

**Review by Andrewpini — APPROVED** [review]

**Comment by alxelax** [comment]
> Do we need a changelog entry for this?

No we don't. We didn't release anything related to nrf54l. Nothing to change.

**Review by mia-ko — APPROVED** [review]

**Pull request merged**